scripts/cnv_wdl/somatic/cnv_somatic_pair_workflow.wdl

```diff
@@ -432,6 +451,23 @@ workflow CNVSomaticPairWorkflow {
                  preemptible_attempts = preemptible_attempts
         }
     }
+    if (select_first([is_run_funcotator, false])) {
+        call CNVFuncotateSegments.CNVFuncotateSegmentsWorkflow as CNVFuncotateSegmentsWorkflow {
+            input:
+                 input_seg_file = CallCopyRatioSegmentsTumor.called_copy_ratio_segments,
```

Comment:
Do you want to expose any more input arguments here as possibilities now?  It seems like it may be useful to add them all up-front.

Reply:
Done